storybook/src/components/form/form.css-template.ts

```diff
@@ -28,6 +28,7 @@ export const cssForm: ComponentImplementation<Form<TemplateResult>> = {
                 : html`${legend}`}
             </legend>
             ${"_$litType$" in formGroups ? formGroups : formGroups.map((formGroup) => formGroupTemplate(formGroup))}
+            ${formButtons ? formButtonsTemplate(formButtons) : ""}
```

Comment:
Als er niets te tonen is dan zetten we geen lege string `""` maar `nothing`, zie andere templates hoe dat werkt.